robottelo/ui/base.py

```diff
@@ -209,6 +212,25 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
         try:
             element = WebDriverWait(
                 self.browser, timeout, poll_frequency
+            ).until(expected_conditions.presence_of_element_located(locator))
+            self.wait_for_ajax(poll_frequency=poll_frequency)
```

Comment:
Will wait_for_ajax really work here? Because if WebDriverWait().until() returns `None` waiting for ajax will not be necessary.

I'm asking to know the reason to use wait_for_ajax.